python/cugraph/cugraph/tests/utils/test_dataset.py

```diff
@@ -81,6 +81,8 @@ def setup(tmpdir):
         dataset.unload()
     gc.collect()
 
+    datasets.set_download_dir(None)
```

Comment:
All these manual cleanups are error-prone (easy to forget). We need to adopt some kind of self-cleaning fixture or context manager instead!